src/main/webapp/index.html

```diff
@@ -1,12 +1,89 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>Nina's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <title>School Form</title>
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
   <body>
-      <p> Hello </p>
+    <p>Rate your classes and professors!</p>
+    <form action="/data" method="POST">
+      <p>Rate your class:</p>
+      <textarea name="class-input">Write opinion on class here.</textarea>
+      <br /><br />
+
+      <p>How would you rate this class?</p>
+      <input
+        type="radio"
+        id="excellent"
+        name="rating-class"
+        value="excellent"
+      />
+      <label for="excellent">Excellent.</label>
+      <input
+        type="radio"
+        id="acceptable"
+        name="rating-class"
+        value="acceptable"
+      />
+      <label for="acceptable">Acceptable.</label>
+      <input type="radio" id="neutral" name="rating-class" value="neutral" />
+      <label for="neutral">Neutral.</label>
+      <input type="radio" id="not-good" name="rating-class" value="not-good" />
+      <label for="not-good">Not good.</label>
+      <input type="radio" id="terrible" name="rating-class" value="terrible" />
+      <label for="terrible">Terrible.</label>
+
+      <br /><br />
+      <textarea name="prof-input">Write opinion on professor here.</textarea>
+      <br /><br />
+
+      <p>How would you rate the professor?</p>
```

Comment:
Not sure if this is this week or next week, but what if there are multiple professors?

Reply:
I was thinking later this week after we talk about parent pointers tomorrow to see whether we can connect the input of the form to all professors of a particular class. Does that sounds goood?